docker/docker_cli.go

```diff
@@ -22,7 +22,9 @@ func DiscoverDockerCli(osWriters logging.OsWriters) (*command.DockerCli, error)
 		return nil, err
 	}
 	opts := &flags.ClientOptions{
-		Common: &flags.CommonOptions{},
+		Common: &flags.CommonOptions{
+			Debug: true,
```

Comment:
I don't think we want this on permanently

Reply:
Correct, I hadn't finished this PR.